src/server/index.ts

```diff
@@ -100,26 +103,43 @@ function startServer() {
   // /create endpoint handles creating a game
   app.post("/create", (req: Request, res: Response) => {
     const body = <CreateGameRequest>req.body;
-    const { wordSet, gameName } = body;
+    const { wordSet, gameName, playerName } = body;
     if (!wordSet) {
       const err = "word set must be defined";
-      res.status(400).send(`{"error":"${err}}`);
+      res.status(400).send(`{"error":"${err}"}`);
       return;
     }
-    if (!gameName) {
-      const err = "game name must be defined";
-      res.status(400).send(`{"error":"${err}}`);
+    for (let i = 0; i < wordSet.length; i += 1) {
```

Comment:
Validate the provided word set (word set is currently autogenerated from a default dict on the client) and game/player names